src/requestVariableCacheValueProcessor.ts

```diff
@@ -0,0 +1,109 @@
+"use strict";
+import { window } from "vscode";
+
+import { MimeUtility } from "./mimeUtility";
+import { RequestVariableCacheValue } from './models/requestVariableCacheValue';
+import { HttpResponse } from './models/httpResponse';
+
+const validPathRegex = new RegExp(/(\w+)(\.\w+|\[\d+\])*/);
```

Comment:
Can we move this syntax check regex directly into the regex `let regex = new RegExp(`\\{\\{\\s*${variableName}(.*)\\s*\\}\\}`, 'g')` of **variableProcessor.ts**, so that we only allow matched request variables to flow into `getValueAtPath ` logic.